cs251tk/student/markdownify/process_file.py

```diff
@@ -28,11 +28,20 @@ def get_file(filename, results, options):
     return True
 
 
-def compile_file(filename, steps, results, supporting_dir):
+def compile_file(filename, steps, results, supporting_dir, basedir, web, student):
+    server_path = ''
+
+    if web:
```

Comment:
Let's just build server_path unconditionally? Since we're just using it to replace the variable?